src/main/java/uk/gov/hmcts/ccd/data/userprofile/UserProfileRepository.java

```diff
@@ -48,7 +52,8 @@ public UserProfileRepository(JurisdictionRepository jurisdictionRepository,
      */
     public UserProfile createUserProfile(UserProfile userProfile, final String actionedBy) {
         if (null != findEntityById(userProfile.getId(), actionedBy, false)) {
-            throw new BadRequestException("User already exists with Id " + userProfile.getId());
+            LOG.warn("User already exists with id:{} so not creating again", userProfile.getId());
+            return userProfile;
```

Comment:
Sateesh, maybe I misunderstood, but I thought we would overwrite in case the user already exists, rather than ignoring the new request.

Reply:
Yeah... that's what PUT is doing atm but it is tightly coupled with workbasket things  :(